
target: ~/.models/goal.sdf
topic_cmd: /cmd_vel
topic_odom: /odom
topic_scan: /scan
threshold_arrive: 0.2
